privval/tcp_test.go

```diff
@@ -25,15 +25,13 @@ func TestSocketPVAddress(t *testing.T) {
 	defer sc.Stop()
 	defer rs.Stop()
 
-	serverAddr := rs.privVal.GetAddress()
+	serverAddr, err := rs.privVal.GetAddress()
+	assert.NoError(t, err)
 
-	clientAddr := sc.GetAddress()
+	clientAddr, err := sc.GetAddress()
+	assert.NoError(t, err)
 
 	assert.Equal(t, serverAddr, clientAddr)
-
-	// TODO(xla): Remove when PrivValidator2 replaced PrivValidator.
-	assert.Equal(t, serverAddr, sc.GetAddress())
```

Comment:
Not sure what this comment means. The above assertion `assert.Equal(t, serverAddr, clientAddr)` should test the same, or not? `sc.GetAddress()` does not change between the two asserts. Hence, removed.